### PR TITLE
add model for install device file

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: lib/cfa/grub2/install_device.rb [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      def self.parse(string)
+        string.lines.map(&:strip).delete_if(&:empty?)
+      end
+
+      # gets list of devices and create file content from it
+      def self.serialize(data)
+        activate = data.delete("activate")
+        generic_mbr = data.delete("generic_mbr")
+
+        res = data.each_with_object("") do |disk, res|
```
Comment (ancorgs) [inline lib/cfa/grub2/install_device.rb]: I find usage of `res` here to both the result and the block variable to be confusing.
What about?
```
res = data.each_with_object("") do |disk, disks|
```
Or `disks_string` or simply `string`... anything but `res`

File: lib/cfa/grub2/install_device.rb [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      # sets special entry activate
+      def activate=(enabled)
+        if enabled
+          return if activate?
+
+          data << "activate"
+        else
+          data.delete("activate")
+        end
+      end
```
Comment (ancorgs) [inline lib/cfa/grub2/install_device.rb]: I'm fine with this code duplication (I'm not obsessed about that). But I fear Rubocop will complain.

File: spec/install_device_spec.rb [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+describe CFA::Grub2::InstallDevice do
+  let(:memory_file) do
+    CFA::MemoryFile.new(
+      "/dev/disk/by-id/ata-TOSHIBA_MG03ACA100_15U5K43SF\n" \
+      "/dev/sdb\n" \
+      "activate\n" \
+      "generic_mbr\n"
+    )
+  end
+  let(:config) do
```
Comment (ancorgs) [inline spec/install_device_spec.rb]: Extremely nitpicking: you could use 
```
subject(:config) do
  ...
```

File: lib/cfa/grub2/install_device.rb [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+      def self.serialize(data)
+        activate = data.delete("activate")
+        generic_mbr = data.delete("generic_mbr")
+
+        res = data.map { |disk| disk + "\n" }
+
+        res << "activate\n" if activate
+        res << "generic_mbr\n" if generic_mbr
+
+        res
```
Comment (ancorgs) [inline lib/cfa/grub2/install_device.rb]: If I'm not mistaken, `#serialize` should return a string, not an array

Comment (jreidinger) [inline lib/cfa/grub2/install_device.rb]: you are right, my fault.

**Comment by ancorgs** [comment]
LGTM

**Pull request merged**